store/server/features/leads/lead-types/mutation.ts

```
@@ -0,0 +1,211 @@
+  }
+};
+
+// --- React Query Mutation Hooks ---
+
+/**
+ * Hook for creating a lead type.
+ */
+export const useCreateLeadType = () => {
+  const queryClient = useQueryClient();
+  return useMutation(createLeadType, {
+    onSuccess: (data) => {
```

ESLint

[error] 'data' is defined but never used.

(@typescript-eslint/no-unused-vars)